tests/prophet/test_prophet_model_export.py

```diff
@@ -348,7 +343,7 @@ def test_log_model_with_extra_pip_requirements(prophet_model, tmpdir):
     # Constraints file
     with mlflow.start_run():
         mlflow.prophet.log_model(
-            prophet_model.model, "model", extra_pip_requirements=[f"-c {req_file.strpath}", "b"]
+            prophet_model.model, "model", extra_pip_requirements=[f"-c {str(req_file)}", "b"]
```

Comment:
```suggestion
            prophet_model.model, "model", extra_pip_requirements=[f"-c {req_file}", "b"]
```

`str` here can be removed.

Reply:
This one still needs a string
```
TypeError: `pip_requirements` must be either a string path to a pip requirements file on the local filesystem or an iterable of pip requirement strings, but got `<class 'pathlib.PosixPath'>`
```

Reply:
We could fix it by updating https://github.com/mlflow/mlflow/blob/7db74ace1aa7e52e271cd00121d25dbafd14386f/mlflow/utils/environment.py#L172-L214 but maybe we do that in another PR?

Reply:
I think `f"-c {req_file}"` is equivalent to `f"-c {str(req_file)}"` if `req_file` is a pathlib.Path object.

```python
>>> import pathlib
>>> p = pathlib.Path("foo")
>>> f"-c {p}"
'-c foo'
>>> f"-c {str(p)}"
'-c foo'
>>> f"-c {p}" == f"-c {str(p)}"
True
```

Reply:
Sorry about that @harupy ! I had accidentally changed the one on top (which wasn't embedded in the f-string). Changing those 12 elements across the 3 tests now!